HG-2791: breaks parquet files into smaller ones

davi-souza opened the pull request.

davi-souza committed:
HG-2791: breaks parquet files into smaller ones

davi-souza committed:
HG-3249: removes harcoded filename

hsyyid committed:
Merge branch 'master' into feature/hg-3249

hsyyid committed:
Bump version to 0.1.0

davi-souza committed:
HG-3249: bumps singer-sdk version and uses 'end of proc' functions

hsyyid committed:
handle missing type

massab-hassan112 committed:
change logger in _parse_timestamps_in_record

massab-hassan112 committed:
add logging for invalid date-time non-string

hsyyid commented on target_parquet/sinks.py:
```diff
@@ -89,11 +88,13 @@ def parse_record_value(record_value, property: dict):
+        try:
+            return datetime_parser.parse(record_value)
+        except Exception as e:
+            logger.warning(f"Could not parse date-time value: {record_value}. Error: {e}")
```
Why did we need to add this `try/except` block?

massab-hassan112 committed:
fix: missing constants for handle_invalid_timestamp_in_record

keyn4 committed:
treating numbers as float64 (#11)

mianazmatali9 committed:
Merge pull request #13 from hotgluexyz/feature/hgi-9174

HGi-9174: Handle empty strings and improve type handling in record parsing

LiorB-D committed:
Revert "Merge pull request #13 from hotgluexyz/feature/hgi-9174" (#14)

This reverts commit 176453e5ba06a3a2226889ef1c05f3c1ce462d0c.

mianazmatali9 committed:
HGi-9174: Handle empty strings and improve type handling in record parsing (#15)

* HGi-9174: Handle empty strings and improve type handling in record parsing

* Allow null values by default if value is empty string

keyn4 committed:
HG-4381 : add file corruption prevention (#12)

keyn4 committed:
bump version (#16)

cesar-lima1 committed:
HGI-9989: add integration tests (#19)

* HGI-9989: add integration tests

* HGI-9989: add GHA for tests

cesar-lima1 committed:
HGI-9989: implement empty_string_to_null config flag (#20)

* HGI-9989: implement empty_string_to_null config flag

- Added a new configuration option `empty_string_to_null` to control whether empty strings are converted to null values.
- Updated the `parse_record_value` function to respect this configuration.
- Added integration tests to verify the behavior of the new configuration option.

* HGI-9989: cursor fix

* HGI-9989: cursor fix 2
